ui/src/Drawer.tsx

```diff
@@ -15,10 +15,27 @@ export class Drawer extends React.PureComponent {
   static contextType = ScholarReaderContext;
   context!: React.ContextType<typeof ScholarReaderContext>;
 
+  addChildClass(pdfViewerContainer: HTMLElement) {
+    // Creating padding for scroll
+    Array.from(pdfViewerContainer.children).forEach(page => {
+      page.classList.add(PDF_VIEWER_DRAWER_OPEN_CLASS)  
+    })
```

Comment:
Hmmm can you point me to the code that suggests they are deleted as they are scrolled out of view? From the behavior I've observed the pages themselves appear to remain and the annotationLayer is deleted so I believe this should still work :). 
<img width="275" alt="Screen Shot 2020-03-05 at 1 25 01 PM" src="https://user-images.githubusercontent.com/22308211/76027041-cc380800-5ee4-11ea-8253-201d3b042486.png">

Reply:
Got it! It's been a while since I've looked closely at page refresh behavior, so I trust your recent observations from your recent investigations 👍  Can you add a comment to this effect just clarifying that page `div`s won't be deleted when pages are re-rendered?